backend/prompt_studio/prompt_studio_index_manager/prompt_studio_index_helper.py

```diff
@@ -1,10 +1,10 @@
 import json
 import logging
-from typing import Optional
 
 from prompt_studio.prompt_profile_manager.models import ProfileManager
 from prompt_studio.prompt_studio_document_manager.models import DocumentManager
 
+from ..prompt_studio_core.exceptions import IndexingAPIError
```

Comment:
@chandrasekharan-zipstack will relative import cause any issue? I remember you mentioning something. If yes, is there anyother way to raise an exception here?

Reply:
I don't think relative imports will cause an error in this module itself but we will have to check docker builds. As a convention I think its best to stick to absolute imports. I'm not able to come up with exact reasons on why it shouldn't be used other than this [SO answer](https://softwareengineering.stackexchange.com/a/159505). @hari-kuriakose your thoughts on this?

Reply:
@gaya3-zipstack There are chances of facing some issues with pre-commit hooks when relative path is used in a file for a class and when the same class is referred in a different file with absolute path. 
Also, I remember @ritwik-g suggestion to use full import paths to avoid build time issues. 
I prefer absolute path over this.

Reply:
Thanks @harini-venkataraman for helping me to recollect. Fixed it.